### PR TITLE
ENG-1159 New carbon OOTB widget entries and assets + homepage page templates updated

### PR DESCRIPTION
**Pull request by jeffgo10**
- added navigation bar (custom element) widget
 - added language chooser (cust el) widget
 - added entando default logo simple widget
 - added keycloak login button (cust el) widget
 - note: did not remove old "keyclock login" widget due to widget specified as default widget
 - removed old navigation bar widget (+1 UX fragment for this widget)
 - new UX Fragment for including transpiled custom element widgets + carbon assets
 - 4 homepage page templates modified: (1) replaced header with header-fragment web component for better handling child widgets (2) <link> css import on carbon components removed due to assets already included in new UX Fragment for Carbon/Widgets (3) body section - the one between header and footer - added min-height

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **alepintus**

If they are not already assigned, you can assign the PR to them by writing `/assign @alepintus` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ichalagashvili — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando-k8s/entando-de-app/pull/98#pullrequestreview-502102067" title="Approved">ichalagashvili</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **alepintus**

If they are not already assigned, you can assign the PR to them by writing `/assign @alepintus` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando-k8s%2Fentando-de-app).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando-k8s/entando-de-app/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
